.github/workflows/tests.yml

```diff
@@ -0,0 +1,16 @@
+name: Run tests
+
+on: [ push, pull_request ]
```

Comment:
What do you think about only running CI on PRs to `main`?

Reply:
I'm fine with it but I think it limits us to only work with branches open against `main`. It may be a few cases where a person is working on something big (like multiple spiders) and want to split it in different PRs...